ocaml/xapi/repository.ml

```diff
@@ -670,6 +673,63 @@ let update_cache ~host ~failed_livepatches =
       ]
       )
 
+let maybe_set_restart_for_all_vms ~__context ~repository_name ~updates_info =
+  let open Guidance in
+  let add_restart_to_vms ~__context ~vms ~kind =
+    let ( let* ) = Option.bind in
+    let* op =
+      match kind with
+      | Mandatory ->
+          Some Db.VM.add_pending_guidances
+      | Recommended ->
+          Some Db.VM.add_pending_guidances_recommended
+      | Full ->
+          Some Db.VM.add_pending_guidances_full
+      | _ ->
```

Comment:
Perhaps be explicit rather than using a wildcard `_`, to be clear about what is ignored. What is the remaining option for `kind`?

Reply:
The only one is just `Livepatch`.